service/templates/layout.html

```diff
@@ -41,7 +41,7 @@
     <div class="phase-banner">
       <p>
         <strong class="phase-tag">BETA</strong>
-        <span>This is a new service - your <a href="mailto:digital-register-feedback@digital.landregistry.gov.uk">feedback</a> will help us to improve it.</span>
+        <span>This service is in private beta. Please give us your <a rel="external" href="mailto:digital-register-feedback@digital.landregistry.gov.uk">Feedback</a>.</span>
```

Comment:
@AndyPorter Adding back in the word private is necessary to satisfy the 90 thing isnt it? Seeing as we aren't in public beta? The question I was raising was that the rest of the wording was reverted too